avtozip/store/templates/store/index.html

```diff
@@ -1,8 +1,21 @@
 {% extends "store/main.html" %}
 
+{% load staticfiles %}
+
 {% block title %}AvtoZip Store{% endblock %}
 
 {% block content %}
+  <link rel="stylesheet" type="text/css" href="{% static 'store/css/main.css' %}" />
   <h1>AvtoZip Store placeholder</h1>
   <p>ProductList placeholder</p>
```

Comment:
Remove this placeholder as you have added the list. Add additional H2 here by now